bigtop-packages/src/common/bigtop-utils/bigtop-detect-javahome

```diff
@@ -36,6 +36,7 @@ OPENJAVA8_HOME_CANDIDATES=(
     '/usr/lib/jvm/java-1.8.0-openjdk-ppc64el'
     '/usr/lib/jvm/java-1.8.0-openjdk'
     '/usr/lib64/jvm/java-1.8.0-openjdk-1.8.0'
+    '/usr/lib/jvm/adoptopenjdk-8-hotspot-amd64'
```

Comment:
`bigtop-detect-javahome` leverages `ls -rvd $JAVA_HOME-CADIDATES` to detect  `JAVA_HOME`.
So pls do not add Arch-tag here (just: `/usr/lib/jvm/adoptopenjdk-8-hotspot` ), otherwise `JAVA_HOME ` couldn't be detected on some platforms (arm64/ppcle).

Reply:
Thanks @guyuqi, I missed that. Will fix it together when rebasing this PR.